test/test_asutil.py

```diff
@@ -56,6 +58,10 @@ def test_content_disposition(self):
         fake_resp = self.fake_resp(suggest_filename="foo.tar")
         self.assertEqual(repour.asutil._find_filename("bar.zip", fake_resp), "foo.tar")
 
+    @pytest.mark.filterwarnings(
+        "ignore:The object should be created within an async function:DeprecationWarning"
+    )
+    # Ignore deprecation warning from third-party library (.tox/py/lib64/python3.11/site-packages/aiohttp/web_app.py)
```

Comment:
Official docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html#pytest-mark-filterwarnings